INSTALLING_PACKAGES.md

```diff
@@ -0,0 +1,27 @@
+# Repository installation
+
+To install only virtualization related packages, for example, you can
+install the open-power-host-os-virt metapackage:
```

Comment:
not related to this patch (maybe you handle it in the following patches, I have not checked them yet):
- I'd first mention how to install a single ordinary package. Start from the more basic info.
- Then, I'd document that several packages may be installed at once by installing a metapackage and list all metapackages and the packages installed by them